controllers/keywordsController.js

```diff
@@ -0,0 +1,25 @@
+const { isValidString, isEmptyString } = require("../utils/validation");
+const keywordModel = require("../models/keywordModel");
+const postModel = require("../models/postModel");
+
+const list = async (req, res) => {
+  if (!isValidString(req.params.keywordId) || isEmptyString(req.params.keywordId)) {
+    return res.status(400).send({ message: "[InvalidKeyword] Error occured" });
```

Comment:
```suggestion
    return res.status(400).send({ message: "[InvalidKeywordId] Error occured" });
```
간단한 수정필요사항 입니다! 검사 데이터와 변수명 맞춰주시면 감사하겠습니다!

Reply:
수정 완료 하였습니다 :)